### ISSUE TITLE
Minor UX issue - empty map when the app is indexing photos for the very first time

### ISSUE THREAD

**Issue by kha84** [deckerst/aves#324]
**Describe the bug**
When you open the app for the very first time, provided you have some huge number of photos, it looks like it's getting indexed somehow to fetch geodata out of shots.
If you miss the notification and go directly to map, you'll see emptiness there, no matter what region of map will you choose (even if you zoom out to see the whole earth). 
I realize it's a small one-off thing, but some users might be frustrated by not seeing their shots on map so it might worth to put some warning text or something saying that the indexing process still ongoing asking user to revisit this view, once the indexing is done.

**To Reproduce**
1. To have some big number of photos in the library 
2. Install Aves and open it for the very first time (or reinstall it / clear cache)
3. Wait until some of your first photos will appear, then tap on top right [...] button, select "Map" item
4. See the map has no items on it and some indexing process just started 

**Expected behavior**
In a map view there should be some text saying "please wait until indexing is over" or something.

**Screenshots**
In this screenshot you see no photos are being shown on map while the indexer process still running. 
![Screenshot_20220913-001410_Aves](https://user-images.githubusercontent.com/110789576/189700962-add870c5-22c3-45e2-a6cb-9a6b1f6bb212.jpg)

**System information and logs:**
Aves version: 1.6.13-play (Build 79)
Flutter version: 3.3.0-0.5.pre (Channel beta)
Android version: 12 (SDK 31)
Android build: SP1A.210812.016.G970FXXSEGVA9
Device: samsung SM-G970F
Mobile services: ready
System locales: en_GB, ru_RU
Aves locale: system -> en
Installer: com.android.vending

**Additional context**
I suppose everything is functioning as designed, so it's more like small feature request. Also you might need to close this map and open it again for your photos to appear (i.e. they're not appearing as they're scanned, but that's ok).

**Comment by deckerst** [comment]
I agree that some kind of in-app feedback could be helpful to fresh users. First impression is especially important.

Actually it's not just the map but everything derived from metadata: countries, tags, motion photos, panoramas, etc.

When there are just a few items to index, I do it in-app, and there's a subtitle in the app bar showing progress. But when there are many items, I do it outside of the app (you saw the running service notification), so that it can continue to index items even when the app is closed. When the service takes care of indexing, I do not relay the indexing progress inside the app. Maybe this would be helpful too.

**Comment by buergi** [comment]
What is worse to me is the fact that you not only have to wait until indexing is done but even have to manually close and reopen the app afterwards to get a populated map. Before restarting all photos which have been opened once are shown, but all others are not. This isaalso described in #606.
